util/fpga/README.md

```diff
@@ -15,9 +15,9 @@ To use this tool flow, we assume you have the following pre-configured.
 
 We use a Docker container to perform cross-compilations of FPGA (Rust) test harness since the FPGA board is running Ubuntu22.04 and may have different system dependency (e.g., glibc) versions than your host workstation.
 
-To build the Docker image run:
+To install the Docker image run:
```

Comment:
It might be nice to also include instructions on how to build the Docker image from source. Is the Dockerfile that the image in the GH registry is built from in the caliptra-sw repo?

Reply:
Agreed, but my preference would be to link to the documentation that lives with the Dockerfile. Since that's missing, I can add it and follow up with a link

Reply:
Where is this Dockerfile being moved to? or does it already exist somewhere else? Also I would prefer we not submit pull requests against my repo fork. I am using this for development and may squash commits from time to time.